infrastructure/nomad/playbooks/templates/jobs/mev-commit-oracle.nomad.j2

```diff
@@ -233,58 +233,6 @@ job "{{ job.name }}" {
             {{ end }}
 
           {% endraw %}
-          {% if env == 'devenv' %}
-          {% raw %}
-          MEV_ORACLE_OVERRIDE_WINNERS=""
-
-          {{- range nomadService "mev-commit-provider-node1" }}
```

Comment:
Why are those providers not needed anymore? Will this work properly for stressnet profile?